packages/toolpad-app/src/toolpad/Apps/index.tsx

```diff
@@ -295,16 +327,18 @@ function CreateAppDialog({ onClose, open, ...props }: CreateAppDialogProps) {
             </Box>
           ) : null}
           {config.recaptchaV2SiteKey ? (
-            <Box
-              id="captcha-target"
-              ref={captchaTargetRef}
-              mt={1}
-              mb={1}
-              sx={{
-                display: 'flex',
-                justifyContent: 'center',
-              }}
-            />
+            <Box sx={{ visibility: requestRecaptchaV2 ? undefined : 'hidden' }}>
```

Comment:
Would this work as a single box?

Reply:
I'm not sure we should add styling to the rendered widget. I feel like it's best to hand over the element to google and handle margin/visibility in a wrapper

Reply:
Ok, I thought the target was the container for the widget and not the widget itself, but if it is we can keep the styles separate, even if just to be safer.

Reply:
updated to be a `div` inside of a `Box`